enterprise/api/throttles.py

```diff
@@ -0,0 +1,29 @@
+"""
+Throttle classes for enterprise API.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from rest_framework.throttling import UserRateThrottle
+
+from django.conf import settings
+
+
+class ServiceUserThrottle(UserRateThrottle):
+    """
+    A throttle allowing the service user to override rate limiting.
+    """
+
+    def allow_request(self, request, view):
+        """
+        Bypass throttling for service user.
+
+        Returns True if the request is coming from the service user, and
+        defaults to UserRateThrottle's configured setting otherwise.
+        """
+        service_users = {
+            getattr(settings, 'ECOMMERCE_SERVICE_WORKER_USERNAME', None),
+            getattr(settings, 'ENTERPRISE_SERVICE_WORKER_USERNAME', None),
+        }
+        if request.user.username in service_users:
+            return True
```

Comment:
I'm going back and forth on whether or not we should eliminate the throttle entirely, or have a second throttle specific to the service user that allows for a greater number of requests/second but is not unbounded.  Thoughts?